src/components/Home.tsx

```diff
@@ -1,41 +1,125 @@
-  typography,
-  TypographyProps,
-  ColorProps,
   SpaceProps,
+  layout,
+  LayoutProps,
+  background,
+  BackgroundProps,
+  FlexboxProps,
+  flexbox,
+  PositionProps,
+  position,
+  ColorProps,
+  color,
+  TypographyProps,
+  typography,
+  border,
+  BorderProps,
 } from "styled-system";
-import { useTranslation } from "react-i18next";
-import i18next from "i18next";
 
-type HeadingProps = ColorProps & SpaceProps & TypographyProps;
+// background
+type BgProps = BackgroundProps & ColorProps & LayoutProps & FlexboxProps;
 
-const Title = styled.h1<HeadingProps>`
-  ${color}
-  ${space}
-  ${typography}
+const Background = styled.div<BgProps>`
+  ${background};
+  ${color};
+  ${layout};
+  ${flexbox};
 `;
 
-const LangButton = styled.button``;
+// logos and button
+type LogoProps = SpaceProps &
+  LayoutProps &
+  FlexboxProps &
+  PositionProps &
+  ColorProps &
+  SpaceProps &
+  TypographyProps &
+  BorderProps;
 
-const Home = () => {
-  const { t } = useTranslation();
-  const onLangClick = useCallback(
-    (countryId: string) => () => i18next.changeLanguage(countryId),
-    []
-  );
+const Logo = styled.a<LogoProps>`
+  ${layout};
+  ${position};
+  ${flexbox};
+  ${space};
+`;
 
+const BuyButton = styled.button<LogoProps>`
+  ${space};
+  ${layout};
+  ${color};
+  ${typography};
+  ${flexbox};
+  ${border}
+  ${position};
+`;
+
+const Home = () => {
   return (
-    <main>
-      <Title color="primary" bg="error" p={3} fontSize={3}>
-        {t("home.header")}
-      </Title>
-      <LangButton onClick={onLangClick("en")}>EN</LangButton>
-      <LangButton onClick={onLangClick("es")}>ES</LangButton>
-    </main>
+    <Background
+      backgroundSize="cover"
+      backgroundImage="url('./assets/home-page/background.jpg')"
+      backgroundPosition="center"
+      backgroundRepeat="no-repeat"
+      bg="black"
+      height="100vh"
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+    >
+      <Logo width={[100, 120, 200, 280]} position="absolute" left={30} top={30}>
+        <img src={oxymore} />
+      </Logo>
+      <Logo width={[100, 200, 300, 400, 500]}>
+        <Link to="/projects" style={{ textDecoration: "none" }}>
+          <img src={alpha} />
+        </Link>
+      </Logo>
+      <Logo
+        width={[40, 40, 60, 70, 80]}
+        position="absolute"
+        left={30}
+        bottom={60}
+        mb={[0, 0, 1, 2, 3]}
+      >
+        <img src={number} />
+      </Logo>
+
+      <BuyButton
+        width={[40, 60, 80, 100, 100]}
+        position="absolute"
+        display="flex"
+        justifyContent="center"
+        fontSize={[1, 1, 1, 3, 3]}
+        font-family="SangBleu OG Serif Light Regular"
+        color="athensGray"
+        py={1}
+        bg="transparent"
+        left={30}
+        bottom={30}
+        border={1}
+        borderColor="athensGray"
+        borderStyle="solid"
+      >
+        BUY
+      </BuyButton>
+      <Logo
+        width={[100, 120, 200, 280]}
```

Comment:
(See image above) there is a little bit extra space at the bottom of this container.
If it `display: flex` and that space will go away

Reply:
Eagle-eyed @mantagen!

Reply:
+ Applied the same to the oxymore logo in the top left